packages/client/examples/browser/storeCar.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<head>
+  <meta charset="utf-8" />
+</head>
+<pre id="out"></pre>
+<script type="module">
+  import { NFTStorage } from 'https://cdn.skypack.dev/nft.storage'
+  import { packToBlob } from 'https://cdn.skypack.dev/ipfs-car/pack/blob'
```

Comment:
am seeing the this get skypack'd to this:
```

'use strict'

const { encoding: getCodec } = require('multibase')
const utf8Decoder = new TextDecoder('utf8')
```
https://github.com/achingbrain/uint8arrays/blob/744af839e7cb3010a2182155822d8b56448a98c4/to-string.js#L1-L16


```
import require$$0 from "/-/multibase@v4.0.4-uhuAWZAoQBdPwI78HUyE/dist=es2020,mode=imports/optimized/multibase.js";
import require$$1 from "/-/web-encoding@v1.1.4-BoaEvv5UU9EtSHTQgnOW/dist=es2020,mode=imports/optimized/web-encoding.js";
const {encoding: getCodec} = require$$0;
const {TextDecoder} = require$$1;
const utf8Decoder = new TextDecoder("utf8");
```
https://cdn.skypack.dev/-/uint8arrays@v2.1.3-MNVirG8yMHFVvhSeyUgN/dist=es2020,mode=imports/unoptimized/to-string.js

Reply:
ah ha! different verison! the actual source of 2.1.3 uses the problematic `web-encoding` package

https://github.com/achingbrain/uint8arrays/compare/v2.1.3...v2.1.5

Reply:
uint8arrays@2.1.3 appears to be the troublesome version at https://github.com/achingbrain/uint8arrays/blob/bf09df0c161ef4d5a951675f0f720ea01832aef5/to-string.js#L4